tests/lib/cli-engine.js

```diff
@@ -643,6 +643,19 @@ describe("CLIEngine", () => {
             assert.equal(report.results[0].messages.length, 0);
         });
 
+        it("should report zero messages when given a config file and a valid file and esprima required as parser", () => {
```

Comment:
I think this unit test is dependent on esprima dn it can break based on changes to esprima (Which is not correct as the functionality this unit test tests is different than testing issue in a code using esprima). Is their another way to test this where its independent of esprima?

Also I would recommend files from `fixtures` dir rather than real cli file as again that file keeps changing.